ra-trustvox/mod.ts

```diff
@@ -0,0 +1,42 @@
+import type { App, AppContext as AC } from "deco/mod.ts";
+import manifest, { Manifest } from "./manifest.gen.ts";
+
+export interface State {
+  /**
+   * @title Store ID
+   * @description Store ID available on the Trustvox dashboard.
+   */
+  storeId: string;
+
+  /**
+   * @title Number of reviews in store carousel.
+   * @description Number of reviews that should appear in the store carousel widget.
+   * @default 7
+   */
+  numberOfReviewsInStoreCarousel?: number;
+
+  /**
+   * @title Enable the staging environment.
+   * @description When enabling the testing environment, the store id must be replaced with a store id from a Trustvox testing environment store.
+   * @default false
+   */
+  enableStaging?: boolean;
+}
+
+export let state: State;
+
+/**
+ * @title RA Trustvox
+ * @description RA trustvox reviews.
+ * @category Review
+ * @logo https://raw.githubusercontent.com/trustvox/deco-apps/enhancement/trustvox-app/ra-trustvox/ra-trustvox.png
+ */
+export default function RATrustvox(
+  props: State,
+): App<Manifest, State> {
+  state = props;
```

Comment:
This WILL not work for all cases, only when the app runs only once with the same props.

One example when this would fail is when previewing the app, you will change the props for the production request while previewing the app.

State exists for that reason, you should access `ctx` instead